rust/protocol/src/lib.rs

```diff
@@ -42,6 +42,7 @@ pub use {
     protocol::{
         CiphertextMessage, CiphertextMessageType, PreKeySignalMessage,
         SenderKeyDistributionMessage, SenderKeyMessage, SignalMessage,
+        CIPHERTEXT_MESSAGE_CURRENT_VERSION,
```

Comment:
I don't think this should be exported. What did you need it for?

Reply:
I have removed this export!